.github/workflows/prebuilt-ld.yml

```diff
@@ -0,0 +1,87 @@
+name: prebuilt-ld
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'prebuilt/ld-*'
+
+jobs:
+  targets:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.targets.outputs.matrix }}
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Create matrix
+        id: targets
+        run: |
+          matrix=$(echo "$(docker buildx bake ld-static-tgz --print | jq -cr '.group.default.targets') $(docker buildx bake ld64-tgz --print | jq -cr '.group.default.targets')" | jq -cs add)
+          echo "matrix=${matrix}" >> ${GITHUB_OUTPUT}
+      -
+        name: Show matrix
+        run: |
+          echo ${{ steps.targets.outputs.matrix }}
+
+  build:
+    runs-on: ubuntu-latest
+    needs:
+      - targets
+    strategy:
+      fail-fast: false
+      matrix:
+        target: ${{ fromJson(needs.targets.outputs.matrix) }}
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Build
+        uses: docker/bake-action@v2
+        with:
+          targets: ${{ matrix.target }}
+          set: |
+            *.output=./dist
+      -
+        name: Move artifacts
+        run: |
+          mv ./dist/**/* ./dist/ || true
+      -
+        name: Upload artifacts
+        uses: actions/upload-artifact@v3
```

Comment:
Why is there this artifact upload/download loop? I don't think we need any artifacts unless there is a release.

Reply:
Indeed not necessary if we are not creating a release